src/js/game_logic.js

```diff
@@ -1,8 +1,28 @@
+const shuffle = (arr1) => {
+    let ctr = arr1.length
```

Comment:
Could this be renamed to something a little more obvious? I'm not sure most will understand just looking at it.

Reply:
Changed to shufflePuzzles

Reply:
The `shuffle` was ok. I meant `ctr`. I assume that's a counter?